reporter/src/main/kotlin/reporters/EvaluatedModelReporter.kt

```diff
@@ -57,12 +59,12 @@ abstract class EvaluatedModelReporter(
     private val serialize: EvaluatedModel.(Writer) -> Unit
 ) : Reporter {
     override fun generateReport(outputStream: OutputStream, input: ReporterInput, options: Map<String, String>) {
-        val start = System.currentTimeMillis()
-        val evaluatedModel = EvaluatedModel.create(input)
-        log.debug { "Generating evaluated model took ${System.currentTimeMillis() - start}ms" }
+        val evaluatedModel = measureTimedValue { EvaluatedModel.create(input) }
```

Comment:
Nice, I was hoping they would add that API because `measureTimeMillis` was often useless but didn't notice that they actually did.